src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -68,7 +68,7 @@ public void TestEncoding(string inputString)
                 }
             }
 
-            Assert.Equal(inString, outString);
+            Assert.True(inString.Equals(outString), $"Encdoing: {Console.InputEncoding}, codepage: {Console.InputEncoding.CodePage}, expected: {inString}, Actual: {outString} ");
```

Comment:
Nit: Encdoing => Encoding
Nit: either capitalize codepage and expected or lowercase Actual, for consistency